src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -99,7 +129,27 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		if (!terminal || !commandDetection) {
 			return;
 		}
-		this._register(commandDetection.onCommandFinished(command => this._resolveQuickFixes(command)));
+		this._register(commandDetection.onCommandFinished(command => {
+			if (this._fixesAvailable) {
```

Comment:
I think we should attach an ID for the quick fix in question, that way we know which ones are successful and which aren't. Just make sure it's a static ID for each quick fix, not the actual command that was run.